integreat_cms/deepl_api/utils.py

```diff
@@ -166,6 +166,30 @@ def deepl_translation(self, request, content_objects, language_slug, form_class)
                     )
                     continue
 
+                if not hix_is_sufficient(source_translation):
+                    messages.error(
+                        request,
+                        _(
+                            'HIX score {:.2f} of "{}" is too low for machine translation (minimum required: {})'
+                        ).format(
+                            source_translation.hix_score,
+                            source_translation.title,
+                            settings.HIX_REQUIRED_FOR_MT,
+                        ),
+                    )
+                    continue
+
+                if getattr(source_translation.foreign_object, "hix_ignore", False):
+                    messages.error(
+                        request,
+                        _(
+                            'Machine translations are disabled for "{}", because its HIX value is ignored'
+                        ).format(
+                            source_translation.title,
+                        ),
+                    )
+                    return False
```

Comment:
```suggestion
                    continue
```
Ah sorry, I messed up my own suggestion :sweat_smile: